public/resources/remix/index.js

```diff
@@ -17,110 +17,127 @@
     'ga(\'send\', \'pageview\');\n' +
     '</script>\n';
 
-    $(head).append(analyticsHtml);
+    head.appendChild(analyticsHtml);
```

Comment:
This seems wrong to me, since `appendChild()` expects a node vs. a string of HTML.  You'll first need to create a `script` element, and then set its content